docs/Velox.md

```diff
@@ -37,9 +48,21 @@ cd /path_to_gluten
 
 ```
 
+<b>For aarch64 build, set the CPU_TARGET to "aarch64":</b>
```

Comment:
Do we need to modify any part of the build script for ARM? e.g. we set the BMI2 flag in our script. Need we remove it?

Can we put the CPU_TARGET as a compile parameter for one_step_veloxbackend.sh and others, not use an env variable?

Reply:
I didn't have to modify any of the build scripts for ARM. I just did a build of this fork today and It passed.

The CPU_TARGET is needed for VELOX build and hence declared it as an ENV variable. We could discover the architecture in the build script and set this variable. I prefer we just leave it as ENV variable because that is what VELOX build does/ expects.